packages/curve-ui-kit/src/hooks/useLocalStorage.ts

```diff
@@ -1,15 +1,15 @@
 import lodash from 'lodash'
-import { SetStateAction, useCallback, useMemo } from 'react'
+import { useMemo } from 'react'
 import type { Address } from '@curvefi/prices-api'
 import type { ColumnFiltersState } from '@tanstack/table-core'
 import type { VisibilityVariants } from '@ui-kit/shared/ui/DataTable/visibility.types'
 import { defaultReleaseChannel, ReleaseChannel } from '@ui-kit/utils'
-import { GetAndSet, type MigrationOptions, useStoredState } from './useStoredState'
+import { type MigrationOptions, useStoredState } from './useStoredState'
 
 const { kebabCase } = lodash
 
 // old keys that are not used anymore - clean them up
-window.localStorage.removeItem('isNewDomainNotificationSeen')
+window.localStorage.removeItem('phishing-warning-dismissed')
```

Comment:
While it's in general good practice to clean things up, I personally don't think it's too necessary. For example, some user might not have `isNewDomainNotificationSeen` cleaned up yet, and it won't ever be cleaned up unless they clear their local browser cache. My point is, even though it's a small thing I personally wouldn't have bothered maintaining a cleanup list. Not a blocker though